### PR TITLE
Linkupdates

### PR DESCRIPTION
**Pull request by stacymcauliffe**
Updated a few items and fixed anchor links for forms documentation. Also moved components and demos used to the bottom of all docs.

### CODE REVIEW

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/alerts-and-notifications/design.md [hunk 1]
```diff
@@ -114,3 +108,8 @@ A more accessible version of this message would look like this:
 ![non-accessible toast notification](img/notification-accessible.png)
 
 While this is more verbose, it has the advantage of being usable for both sighted and non-sighted users by providing instructions to screen reader users about where they can go to complete the action.
+
+## Related components and demos
+**Core HTML/CSS:** [Alert](/documentation/core/components/alert), [Alert Group](/documentation/core/components/alert-group), Alert Demo
+
+**React:** [Alert](/documentation/react/components/alert), [Alert Group](/documentation/core/react/alert-group)
```
Comment (jschuler) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/alerts-and-notifications/design.md]: This link doesn't look quite right, `/core/react/`

**Comment by stacymcauliffe** [comment]
Good call @jschuler - I actually removed that component bc I am not seeing it in the system. Also removed "Alert demo" since I don't see that yet either.

File: packages/patternfly-4/content/design-guidelines/usage-and-behavior/about-modal/design.md [hunk 2]
```diff
@@ -18,7 +18,10 @@ The About Modal stores the version number(s) of the application as well as any a
 7. Close
 8. Modal image: The modal image should be the same as the [background image](/documentation/react/components/backgroundimage) you use for your application.
 
-## Components and demos used
+## Usage
+Use the About Modal to provide information about the product including the product name, logo, version numbers, and any appropriate legal text.
+
+## Related components and demos
```
Comment (mcarrano) [inline packages/patternfly-4/content/design-guidelines/usage-and-behavior/about-modal/design.md]: I didn't understand what we had here. Components and Related component? Seems redundant. Maybe just an editing error?

Comment (stacymcauliffe) [reply packages/patternfly-4/content/design-guidelines/usage-and-behavior/about-modal/design.md]: @mcarrano - I think I fixed it

**Pull request merged**